### DISCUSSION TITLE
thanks for this great plugin! Future update feature request - toolbar button

### DISCUSSION THREAD

**Discussion by greatpotoo** [Leseratte10/acsm-calibre-plugin#36] [Ideas]
Dear @Leseratte10,

**thanks** for this great and very useful plug-in which makes the management of borrowed ebooks so much easier.

I was hoping to place a small feature request for a future update:
Would it be possible to create a toolbar button for the "show loaned books" menu, this would allow integration into the [main] toolbar for quick and easy access to the loaned books list and provide the ability to quickly return books that have been read instead of having to call up the plug-ins menu via preferences and going through 4 steps before being able to access the list of loaned books.

Thanks for your consideration!

**Comment by Leseratte10** [comment]
Hi @greatpotoo 
this feature is already implemented and will be available with the next update (v0.0.17). 

If you want, you can already download the test version from https://github.com/Leseratte10/acsm-calibre-plugin/suites/8142173065/artifacts/353076906 but I'd recommend to make a backup of your authorization data before, as this version changes how that is stored and without the backup it might not be easily possible to downgrade back to 0.0.16 or earlier.

**Reply by greatpotoo** [reply]
thanks for your quick response @Leseratte10!

the test version works great!

**Reply by Armandprogame** [reply]
Hello Leseratte, the Wiimmfi priioader patch for Wii U also includes patching the GameCube controller adapter or is that a separate thing?

**Reply by Armandprogame** [reply]
Hi @Leseratte10, the new build 21 of the wiimmfi competition patch can you release it for everyone? 
Or are there still problems?

**Reply by Leseratte10** [reply]
That will be release when it's ready and when I have time. Please stop using this github repo for things unrelated to the plugin.
